### ISSUE TITLE
Enhancement - Max indentation on replies

### ISSUE THREAD

**Issue by 1dolinski** [lml/commontator#143]
Additional config setting for reply (config setting :i) It would be nice to have the option to limit the number of indentations in config, similar to YouTube

Replying to any comment just keeps the indentation at 1
![image](https://user-images.githubusercontent.com/1243216/64141868-bc295980-cdd7-11e9-8272-f517629eac53.png)

Coming back to page, replies are compressed 
![image](https://user-images.githubusercontent.com/1243216/64141876-bfbce080-cdd7-11e9-897d-45da0ce9a5eb.png)

**Comment by Dantemss** [comment]
Limiting indentation would certainly be interesting.
You can get something similar to the reply compression by playing with the `comments_per_page` setting. Make it an array with only 1 element for example. It's just a little funky because currently Commontator does not support a "load more comments" (or "view x replies")-style pagination.

**Comment by 1dolinski** [comment]
Definitely, it seems the limited indentation would make the comment feed more mobile-friendly.
